test/celeritas/ext/GeantImporter.test.cc

```diff
@@ -1293,6 +1309,89 @@ TEST_F(OneSteelSphereGG, physics)
     }
 }
 
+TEST_F(LarSphere, optical)
+{
+    auto&& imported = this->imported_data();
+    EXPECT_EQ(1, imported.optical.size());
+
+    // First material is vacuum, no optical properties
+    MaterialId vacuum_id{0};
+    EXPECT_EQ("vacuum", imported.materials[vacuum_id.get()].name);
+    auto const vacuum_iter = imported.optical.find(vacuum_id.get());
+    EXPECT_TRUE(vacuum_iter == imported.optical.end());
+
+    // Second material is liquid argon
+    MaterialId lar_id{1};
+    EXPECT_EQ("lAr", imported.materials[lar_id.get()].name);
+    auto const lar_iter = imported.optical.find(lar_id.get());
+    ASSERT_FALSE(lar_iter == imported.optical.end());
+    auto const& optical = lar_iter->second;
+
+    // Check optical material ID
+    auto materials = MaterialParams::from_import(imported);
+    ASSERT_TRUE(materials);
+    EXPECT_EQ(OpticalMaterialId{},
+              materials->get(vacuum_id).optical_material_id());
+    EXPECT_EQ(OpticalMaterialId{0},
+              materials->get(lar_id).optical_material_id());
+
+    real_type const tol = this->comparison_tolerance();
+
+    // Check scintillation optical properties
+    auto const& scint = optical.scintillation;
+    EXPECT_TRUE(scint);
+    EXPECT_EQ(1, scint.resolution_scale);
+    EXPECT_EQ(5, scint.yield);
```

Comment:
Is this yield/MeV?  If so, may this be changed 50000 to be realistic for LAr (i.e, from input gdml)?